app/javascript/components/Pages/Recognition.js

```diff
@@ -51,6 +61,10 @@ const Recognition = ({data, setData, saveDataToDb, steps, service}) => {
           <p className='fw-semibold mb-0  pt-1 pb-1 cut-text'>{parse(shoutOut.rich_text)}</p>
         </span>
         <img  id={ shoutOut.id } src={edit_pencil} alt="pencil" className='pencil' onClick={editHandling}/>
+        <span className="expand-link" >
+          <img  src={trash} alt="trash" className='trash' onClick={trashHandling}/>
+          <img  id={ 'trashRed'+shoutOut.id  }src={trashRed} alt="trash" className='trashRed' onClick={trashHandling}/>
```

Comment:
be careful with spaces in tags components. Space before 'src' is missing

Reply:
Thanks